services/explorer-indexer/src/modules/syncer/syncer.module.ts

```diff
@@ -24,6 +25,17 @@ import { ConnectorsModule } from '../connectors';
         };
       },
     }),
+    BullModule.registerQueue(
+      {
+        name: 'pbfts',

```

Comment:
This will be the historical sync of the pbfts, right? We should add historical-pbfts as name. Same for dags.